pkg/testgridanalysis/testidentification/component.go

```diff
@@ -136,6 +136,7 @@ func init() {
 	must(addSigMapping("sig-imageregistry", "Image Registry"))
 	must(addSigMapping("sig-network", "Networking"))
 	must(addSigMapping("sig-node", "Node"))
+	must(addSigMapping("sig-openshift-logging", "Logging"))
```

Comment:
this is the sig name that was chosen, and not `sig-logging`?
(if so this seems fine, just want to confirm)

Reply:
In [DPTP-1691](https://issues.redhat.com/browse/DPTP-1691?focusedCommentId=15378281&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15378281), I was suggested to use `sig-openshift-logging` as the sig name.